pkg/pillar/types/locationconsts.go

```diff
@@ -4,21 +4,44 @@
 package types
 
 const (
-	appImgObj = "appImg.obj"
-
-	// TmpDirname - Temporary dir for zededa components
+	// TmpDirname - temporary dir. for agents to use.
 	TmpDirname = "/var/tmp/zededa"
-	// PersistDir - Persist Directory
+
+	// PersistDir - Location to store persistent files.
 	PersistDir = "/persist"
-	// PersistRktDataDir - persist rkt dir used by rkt
-	PersistRktDataDir = PersistDir + "/rkt"
-	// RwImgDirname - location for images
-	RwImgDirname = PersistDir + "/img" // We store images here
-	// DownloadDirname - Location for downloader to download images
+	// DownloadDirname - Location of downloaded images
 	DownloadDirname = PersistDir + "/downloads"
-	// ImgCatalogDirname - Location for App Images
-	ImgCatalogDirname = DownloadDirname + "/" + appImgObj
-	// VerifiedDirname - Read-only images named based on sha256 hash each
-	// in its own directory
+	// ObjectDownloadDirname - Location of downloaded objects
+	ObjectDownloadDirname = PersistDir + "/downloads"
```

Comment:
On line 13 you have a different name for the same directory. There should be only one name for each directory.

Reply:
Hmm.. Thinking was.. The same directory could be used for different purposes.. So could use different names.. Fixed it by doing:
   ObjectDownloadDirname = DownloadDirname

Will look through some of these usages to see if it good to replace ObjectDownloadDirname with DownloadDirname...